public/docs/ts/latest/cookbook/graphql.jade

```diff
@@ -0,0 +1,688 @@
+  * [Apollo Developer resources](http://dev.apollodata.com/) - The [team](http://www.apollodata.com/) behind the Angular GraphQL client,
+  there you will find a more advanced resources about:
+    * [Handling updates from the server and managing the local store](http://dev.apollodata.com/angular2/receiving-updates.html)
+    * [Authentication](http://dev.apollodata.com/angular2/auth.html)
+    * [Pagination](http://dev.apollodata.com/angular2/pagination.html)
+    * [Server-side rendering](http://dev.apollodata.com/angular2/server-side-rendering.html)
+    * and more..
+  * [Apollo Dev Blog](https://dev-blog.apollodata.com/) - The most popular GraphQL blog
+  * [Apollo Client Developer Tools](https://dev-blog.apollodata.com/apollo-client-developer-tools-ff89181ebcf#.n5f3fhbg2) - GraphQL debugging tools for Apollo Client in the Chrome developer console
+
+.l-main-section
+<a id="server"></a>
+:marked
+  ## Appendix: Setting up a GraphQL server
+
+  Running the server in the browser helps you start using 
+  GraphQL on your frontend without having a GraphQL backend. 
+  If you don't have the option of running GraphQL on the server, 
+  this method makes it possible to still use GraphQL in your app with the
+  benefits for not needing to sync multiple REST requests and join logic 
+  on the client.
+
+.alert.is-important
+  :marked
+    Uri: Can you tell me why it makes it easier? We should say 
+    "...logic on the client because..."
+    :Kapunahele - changed above
+
+
+.l-sub-section
+  :marked
+    To read more about how to run a full GraphQL backend, see the [Apollo Server documentation](http://dev.apollodata.com/tools/).
+    Because the real, backend server is written in Isomorphic Javascript, 
+    it is almost identical to the local server in this appendix.
+    Everything you learn here applies to writing an actual GraphQL backend server.
+
+    Additionally, there are a few GraphQL backend-as-a-service platforms available, 
+    similar to Firebase, but based on the GraphQL API spec.
+    For help on getting up and running, see [Scaphold](https://www.scaphold.io/) and [Graphcool](https://www.graph.cool/).
+
+.alert.is-important
+  :marked
+    Uri: Can you explain why the GraphQL API makes using things like Firebase different? 
+    There's a "but" in that sentence, but I don't know what it implies.
+    :Kapunahele - the word `but` was wrong. fixed and edited a bit
+
+:marked
+  In order to create a GraphQL schema, you need the `graphql-tools` library.
+  It allows you to write a GraphQL schema as a string and make it executable. 
+  In a terminal window, issue the following command:
+
+code-example(language="sh" class="code-shell").
+    npm install graphql-tools --save
+
+:marked
+  Next, create a file called `in-memory-graphql.ts` in the `app` directory 
+  and paste in the following schema:
+
++makeExample('heroes-graphql/ts/src/app/in-memory-graphql.ts', 'graphql-schema', 'in-memory-graphql.ts')
+
```

Comment:
Please make sure the comments are complete sentences. Here are my recommended changes:

> The schema allows the following queries:

> This schema allows the following mutation:

> ~~you need to~~ Tell the server which types represent the root query
> and root mutation types. By convention, they are called RootQuery and RootMutation.

Next, I would add a couple of sentences that are basically echoing what the comments say so that the prose is complete. We want everything touched on in the prose and comments to serve as a way to drive home what we talk about in the prose.